demo/scripts/controls/sidePane/editorOptions/getDefaultContentEditFeatureSettings.ts

```diff
@@ -2,10 +2,14 @@ import { ContentEditFeatureSettings } from 'roosterjs-editor-types';
 import { getAllFeatures } from 'roosterjs-editor-plugins/lib/ContentEdit';
 import { getObjectKeys } from 'roosterjs-editor-dom';
 
-export default function getDefaultContentEditFeatureSettings() {
+export default function getDefaultContentEditFeatureSettings(): ContentEditFeatureSettings {
     const allFeatures = getAllFeatures();
-    return getObjectKeys(allFeatures).reduce((settings, key) => {
-        settings[key] = !allFeatures[key].defaultDisabled;
-        return settings;
-    }, <ContentEditFeatureSettings>{});
+    return {
+        ...getObjectKeys(allFeatures).reduce((settings, key) => {
+            settings[key] = !allFeatures[key].defaultDisabled;
+            return settings;
+        }, <ContentEditFeatureSettings>{}),
+        indentWhenAltShiftRight: true,
```

Comment:
Why change this?

Reply:
To always have these features enabled in the demo page. 
We added defaultDisabled on the Alt+Shift+Right/Left ContentEdit Features when Browser is in Mac.

Reply:
We can remove it so we have these features disabled in Mac if needed.

Reply:
I think it is ok, since it only in demo site.

Reply:
Thanks